src/components/Disclaimer/Consent.tsx

```diff
@@ -42,35 +42,43 @@ export const Consent: FC = () => {
         <PurpleModalContent>
           <div className="w-full max-w-prose text-pink-100 max-md:text-xs md:leading-relaxed">
             <p className="mb-3">
-              Fortress Finance is a set of smart contracts made available by
-              Fortress.Finance on a voluntary, “as-is” and “as available” basis.
-              By interacting or attempting to interact (in both cases,
-              “interacting”) with Fortress Finance, you confirm that you
-              understand and agree to these terms:
-            </p>
-            <p className="mb-3">
-              Nature of Fortress Finance: The protocol is completely
-              decentralized, Fortress.Finance is not involved in any
-              transactions, whether as an intermediary, counterparty, advisor or
-              otherwise.
+              Fortress Finance is a web-based user interface that enables you to
+              interact with a set of decentralised autonomous smart-contract
```

Comment:
This sentence doesn't quite make sense. 

It should say `...enables you to interact with a decentralized, autonomous smart-contract system deployed on EVM-compatible blockchain networks.`